content.js

```diff
@@ -2718,37 +2710,48 @@ async function ObserverBattle() {
     }
 }
 async function ObserverImage() {
-    var allElements = doc.querySelectorAll('[class^="contents"]')[0];
-    // var allElements = doc.getElementById('wrapper');
+    var allElements = doc.getElementById('wrapper');
     if (!allElements) {
         //The node we need does not exist yet.
         //Wait 500ms and try again
         window.setTimeout(ObserverImage, generalConfig.refreshRate);
         return;
     }
-    if (doc.URL.includes('#raid')) {
-        window.setTimeout(ObserverImage, generalConfig.refreshRate);
+    if (doBattleTrans) {
+        if(doc.URL.includes('#raid')){
+            ImageObserver.disconnect();
+            window.setTimeout(ObserverImage, generalConfig.refreshRate);
+            return;
+        }
+    }
+    if(doc.URL.includes('#raid')){
+        archiveObserver.disconnect();
+        window.setTimeout(ObserverArchive, generalConfig.refreshRate);
         return;
     }
-    ImageObserver.observe(allElements, config);
-    ImageObserver.observe(doc.querySelectorAll('[class^="pop-global-menu"]')[0], config); // Upper menu
+    ImageObserver.observe(allElements, config_image);
 }
 async function ObserverImageDIV() {
-    var allElements = doc.querySelectorAll('[class^="contents"]')[0];
-    // var allElements = doc.getElementById('wrapper');
+    var allElements = doc.getElementById('wrapper');
     if (!allElements) {
         //The node we need does not exist yet.
         //Wait 500ms and try again
-        window.setTimeout(ObserverImageDIV, generalConfig.refreshRate);
+        window.setTimeout(ObserverImage, generalConfig.refreshRate);
         return;
     }
-    if (doc.URL.includes('#raid')) {
-        window.setTimeout(ObserverImageDIV, generalConfig.refreshRate);
+    if (doBattleTrans ) {
+        if(doc.URL.includes('#raid')){
+            ImageObserver.disconnect();
+            window.setTimeout(ObserverImageDIV, generalConfig.refreshRate);
+            return;
+        }
+    }
+    if(doc.URL.includes('#raid')){
+        archiveObserver.disconnect();
+        window.setTimeout(ObserverArchive, generalConfig.refreshRate);
```

Comment:
```suggestion
    if(doc.URL.includes('#raid')){
        ImageObserver.disconnect();
        archiveObserver.disconnect();
        window.setTimeout(ObserverImageDIV, generalConfig.refreshRate);
```

전투 url 일때는 작동시키지 않음.